### ISSUE TITLE
Settings editor - TOC and settings list tab order reversed

### ISSUE THREAD

**Issue by roblourens** [Microsoft/vscode#55799]
- Focus a setting
- tab
- TOC is focused

It should be left -> right

This came from changes to get the shadow to be full width. If the TOC comes first in the DOM, then the settings' ScrollableElement intercepts scroll events over the TOC. So I reversed the DOM order but there's probably a better fix. Or, we get rid of the shadow.
